scorelab/dengue-stop
# Configure Redux

## What did you implement:

Closes #76 

## How did you implement it:

- Implemented the initial store for Redux with one reducer and one action for the `home` component.
- Added the Redux store wrapper to the AppNavigator.js file, so that the store will be accessible all the components in the app.
- Wired up the `home` component to the Redux store as a container.
- Implemented the use case of Redux for the `home` component. Basically it will fetch the value for "TOTAL NUMBER OF REPORTED CASES" from the Redux store and will update the value via dispatching a Redux action inside the `componentDidMount` life-cycle hook.

## How can we verify it:

Skip the `Login screen` and navigate to the `Home screen`. There you will see that the value for "TOTAL NO OF REPORTED CASES" gets change to 1000 after a few seconds which has a initial value of 67456.
![image](https://user-images.githubusercontent.com/15249242/37246235-ab046fca-24cb-11e8-93c4-186722b4bb75.png)


## Todos:

- [ ] Configure redux-persist for storage hydration
- [ ] Configure redux-saga to support the Saga Side Effects Model

***Is this ready for review?:*** YES
***Is it a breaking change?:*** YES
